### PR TITLE
Mr order controller

### PR DESCRIPTION
**Pull request by MattRowee**
# Description

-User should be able to GET a list, and GET a single Order.
When an order is deleted, every line item (i.e. entry in OrderProduct) should be removed
-Using the `?completed=false` query string parameter should return Orders that with a 'NULL' PaymentId.
-Using the query string parameter of `?include=product` should return a list of products with each order
- Using the query string parameter of `?include=customer` should return the customer object associated with the order

##TESTING CRITERIA
git add and git commit what you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout mr-order-controller to access my branch.
If you don't see the most recent version of their files, try git pull origin mr-order-controller
Run your server on my branch and make sure everything looks okay.
Once server is running, use your browser to navigate to api/orders to ensure correct information is displayed
Afterwards, in VisualStudio, go to the OrderTest.cs file under T TestBangazonAPI and right-click on it and click on Run Tests. Ensure that all tests come back green.

Check verbiage and comments to make sure they are legible and understandable.

- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [x ] My changes generate no new errors
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by BatsGoScreechh** [comment]
Everything works! Code is commented properly. 👍

File: BangazonAPI/BangazonAPI/Models/PaymentType.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class PaymentType
         public string name { get; set; }
         public int accountNumber { get; set; }
         public int CustomerId { get; set; }
-        public Customer customerPaying { get; set; } = new Customer();
+        //public Customer customerPaying { get; set; }
```
Comment (ColonelSped) [inline BangazonAPI/BangazonAPI/Models/PaymentType.cs:14]: Does this need to be deleted? We don't quote out lines of code.

**Review by ColonelSped — CHANGES_REQUESTED** [review]

**Comment by ColonelSped** [comment]
Instructions need to state customer and product not customers and products.

File: BangazonAPI/BangazonAPI/Models/PaymentType.cs (hunk 1 above)
Comment (MattRowee) [reply BangazonAPI/BangazonAPI/Models/PaymentType.cs:14]: cut it out, good to go

**Review by ColonelSped — APPROVED** [review]

**Review by ColonelSped — APPROVED** [review]

**Comment by MattRowee** [comment]
WOW, MASTERFUL JOB, GENIUS CODING

**Comment by ColonelSped** [comment]
👍

**Pull request merged**